client/src/App.jsx

```diff
@@ -0,0 +1,132 @@
+import React, { useState } from "react";
+import WeatherCard from "./components/WeatherCard";
+import CircularProgress from "@mui/material/CircularProgress";
+import TextField from "@mui/material/TextField";
+import Button from "@mui/material/Button";
+import Container from "@mui/material/Container";
+import Paper from "@mui/material/Paper";
+import Typography from "@mui/material/Typography";
+
+import "./App.css";
+
+function App() {
+  const [city, setCity] = useState("");
+  const [weatherData, setWeatherData] = useState("");
+  const [error, setError] = useState(null);
+  const [loading, setLoading] = useState(false);
+
+  const sharedCSS = {
```

Comment:
I know I said "shareCSS" in my comment, but that was just for understanding purposes. I think a better more relevant name like "paperStyles" would be more appropriate, what do you think?

Reply:
That does sound like a more useful name!